src/com/carolinarollergirls/scoreboard/utils/ScoreBoardClock.java

```diff
@@ -0,0 +1,66 @@
+package com.carolinarollergirls.scoreboard.utils;
+
+import com.carolinarollergirls.scoreboard.defaults.DefaultScoreBoardEventProvider;
+import com.carolinarollergirls.scoreboard.event.ScoreBoardEvent;
+
+public class ScoreBoardClock extends DefaultScoreBoardEventProvider{
+	private ScoreBoardClock() {
+		offset = System.currentTimeMillis();
+	}
+	
+	public String getProviderName() { return "ScoreBoardClock"; }
+	public Class<ScoreBoardClock> getProviderClass() { return ScoreBoardClock.class; }
+	public String getProviderId() { return ""; }
+
+	public static ScoreBoardClock getInstance() {
+		if (instance == null) {
```

Comment:
If you're doing this you need locking.

A static initialiser might be simpler.